City-of-Helsinki/parking-permits
# Add PERMIT_EXTENSIONS_ENABLED flag to allow extending permits

## Description

Adds an environment variable `PERMIT_EXTENSIONS_ENABLED` which disables this feature by default.

## Context

[PV-794](https://helsinkisolutionoffice.atlassian.net/browse/PV-794)

## How Has This Been Tested?

Unit and manually tested 

## Manual Testing Instructions for Reviewers

Webshop will no longer show "extend permit" button.

Set PERMIT_EXTENSIONS_ENABLED=true in local env to re-enable.



[PV-794]: https://helsinkisolutionoffice.atlassian.net/browse/PV-794?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
